src/Espell.package/EPBackend.class/instance/fullBlockClass..st

```diff
@@ -0,0 +1,4 @@
+special objects
+fullBlockClass: aClass
+
+	
```

Comment:
Is this method really needed?

Reply:
I would say yes, it is called from the changes I did in PharoBootstrap. Obviously this method is in an abstract class and it is called from another abstract class, while the subclasses have different concrete versions, so it is not really needed in the sense that only concrete subclasses are used. I think it's better to have it for future compatibility, but I can remove it if you want to.

Reply:
Ok, we see it later. So far the other backend (the one that uses the old oz vm) is not used anymore...